test/e2e/app-dir/ppr-metadata-streaming/app/dynamic-page/page.tsx

```diff
@@ -0,0 +1,31 @@
+import { cookies } from 'next/headers'
+import { Suspense } from 'react'
+import Link from 'next/link'
+
+// Dynamic usage in page, wrapped with Suspense boundary
+export default function Page() {
```

Comment:
I'm seeing a hydration error on `next start` when I load this page that I think is worth investigating

Reply:
seems caused by this Suspense boundary, but not sure why this is not erroring in dev 🤔 
```
<Suspense >
  <SubComponent />
</Suspense>
```

Reply:
add new assertions to ensure there's no console error in browser